FirstLesson/DrawEllipseAndPoly/DrawEllipseAndPoly.cpp

```diff
@@ -0,0 +1,76 @@
+//
+// Created by s1000633 on 5/13/21.
+//
+#include <opencv2/opencv.hpp>
+using namespace cv;
+#define w 512
```

Comment:
please provide a more descrptive name for variable w,
also instead of define you could use constexpr instead of macros